### ISSUE TITLE
pldoc issue with latest swipl-devel (7.5.13-7-gf9d7db01a)

### ISSUE THREAD

**Issue by jrvosse** [ClioPatria/ClioPatria#57]
Jan,

A clean cliopatria install returns a 500 server error with error message:
Internal server error
http_alias `pldoc_pkg' does not exist

Setting 'doc_enable(true)' seems to solve the issue.  Is this something that needs to be set in the default Clioptria configuration?

**Comment by JanWielemaker** [comment]
Yes, that must be changed added to cliopatria startup code, but conditional so we do not force the latest Prolog.  Shall I do so?

**Comment by jrvosse** [comment]
yes please! thanks!
